docs/security/host-certs.md

```diff
@@ -60,20 +60,64 @@ If your institution is not in the list and Let's Encrypt certificates do not mee
 
 1. Generate a Certificate Signing Request (CSR) and private key:
 
+1. Create a custom openssl config file for this, `incommon.conf`, using your institution's information for the
+   different fields in the `[ dn ]` section, and the desired hostname for the `CN` field.
+
+        :::console
+        [ req ]
+        default_bits = 2048
+        prompt = no
+        encrypt_key = no
+        default_md = sha256
+        distinguished_name = dn
+
+        [ dn ]
+        DC = org
+        DC = incommon
+        C = %RED%US%ENDCOLOR%
+        ST = %RED%Wisconsin%ENDCOLOR%
+        L = %RED%Madison%ENDCOLOR%
+        O = %RED%University of Wisconsin-Madison%ENDCOLOR%
+        CN = %RED%voms.opensciencegrid.org%ENDCOLOR%
+
+
+1. If you need to generate a CSR with Subject Alternative Names (SAN), add these lines to your `incommon.conf`,
+   with as many alternate DNS entries as needed in the `[ alt_names ]` section:
+
+        :::console
+        [ req ]
+        req_extensions = req_ext
+
+        [ req_ext ]
+        subjectAltName = @alt_names
+
+        [ alt_names ]
+        DNS.1 = %RED%server1.example.com%ENDCOLOR%
+        DNS.2 = %RED%mail.example.com%ENDCOLOR%
+        DNS.3 = %RED%www.example.com%ENDCOLOR%
+        DNS.4 = %RED%<SAN 4>%ENDCOLOR%
+        DNS.5 = %RED%<SAN 5>%ENDCOLOR%
```

Comment:
This doesn't really show that you can have any number of SANs. How about something like this?

```
        %RED%DNS.1 = server1.example.com%ENDCOLOR%
        %RED%DNS.2 = <SAN 2>%ENDCOLOR%
        [...]
        %RED%DNS.N = <SAN N>%ENDCOLOR%
```

Reply:
Ah, ok.  I had hoped that the language

> with as many alternate DNS entries as needed in the `[ alt_names ]` section:

would make that clear, but sure we can use the ellipses and `<SAN N>`

Are you sure you prefer to make the lefthand side red also?

Reply:
It doesn't hurt to double down on our clarity here.

And yea, I think making the LHS red, too, makes sense. I reserve the right to change my mind after I see it, though :)